id/community/podcasts/index.md

```diff
@@ -0,0 +1,26 @@
+---
+layout: page
+title: "Podcasts"
+lang: id
+---
+
+Dengarkan berita, wawancara, dan diskusi terkait Ruby dan komunitasnya.
+
+[Ruby Rogues][rogues]
+: Ruby Rogues podcast adalah sebuah diskusi panel terkait topik yang
+  berhubungan dengan pemrograman, karier, komunitas, dan Ruby.
+
+[Ruby on Rails Podcast][rorpodcast]
+: Ruby on Rails Podcast, sebuah pembicaraan mingguan terkait Ruby on Rails,
+  perangkat lunak *open source*, dan profesi pemrograman.
+
+### Terlibat
```

Comment:
I think it's better to use word "Ikut Berkontribusi" or "Kontribusi"

Reply:
Ah, that's a good suggestion. I prefer "Ikut Berkontribusi". @kuntoaji 

Just updated.